### PR TITLE
Cover Errata Widget for Filtered User (BZ1417114)

### PR DESCRIPTION
**Pull request by oshtaier**
Functionality works properly for 6.2.8, but is still in POST for 6.3, so gonna fail on last statement:
```
self.assertIsNotNone(self.dashboard.wait_until_element(
    locators['dashboard.latest_errata.empty']))
```
because of:
image

```
nosetests tests/foreman/ui/test_dashboard.py -m test_positive_user_access_with_host_filter
S
----------------------------------------------------------------------
Ran 1 test in 12.151s

OK (SKIP=1)
```

### CODE REVIEW

File: tests/foreman/ui/test_dashboard.py [hunk 1]
```diff
@@ -713,6 +714,66 @@ def test_positive_host_collections(self):
...
+        """
+        user_login = gen_string('alpha')
+        user_password = gen_string('alphanumeric')
+        org = entities.Organization().create()
+        # create a role with necessary permissions
+        role = entities.Role().create()
+        entities.Filter(
+            permission=entities.Permission(
+                resource_type='Organization').search(
+                filters={'name': 'view_organizations'}),
```
Comment (abalakh) [inline tests/foreman/ui/test_dashboard.py]: Have you tried `query={'name': 'view_organizations'}` or `query={'search': 'name=view_organizations'}` instead? Filters should be a last resort, as they will grab a list of entities, execute `read()` for each single one and only then form a result list.
tl;dr - filters generate a lot of extra `read()` operations decreasing overall performance, try to avoid them if that's possible.

**Comment by oshtaier** [comment]
About xpath:
in case of error - it would be span
in case of content - it would be tr
in case of empty string - it would be p. It doesn't make sense to fetch data from p as it cannot be reused in any way except current situation, so made decision to not follow proposed idea(which of course I thought about in the past)

File: tests/foreman/ui/test_dashboard.py (hunk 1 above)
Comment (abalakh) [inline tests/foreman/ui/test_dashboard.py]: Oh, and a third option (the most obvious one though it works pretty rarely): `entities.Permission(name='view_organizations', resource_type='Organization').search()`
